bundles/org.openhab.binding.fsinternetradio/src/main/java/org/openhab/binding/fsinternetradio/internal/FSInternetRadioDiscoveryParticipant.java

```diff
@@ -89,44 +95,49 @@
         // and: https://community.openhab.org/t/internet-radio-i-need-your-help/2131/20
         // and: https://community.openhab.org/t/internet-radio-i-need-your-help/2131/23
         // these radios do not provide model number, but the model name should also be ok
-        final Set<String> radiosWithoutManufacturer = new HashSet<String>();
+        final Set<@Nullable String> radiosWithoutManufacturer = new HashSet<@Nullable String>();
         radiosWithoutManufacturer.add(""); // empty manufacturer / model name
         radiosWithoutManufacturer.add(null); // missing manufacturer / model name
         SUPPORTED_RADIO_MODELS.put("SMRS18A1", radiosWithoutManufacturer);
         SUPPORTED_RADIO_MODELS.put("SMRS30A1", radiosWithoutManufacturer);
         SUPPORTED_RADIO_MODELS.put("SMRS35A1", radiosWithoutManufacturer);
 
+        // as reported on https://community.openhab.org/t/controlling-teufel-3sixty/88690/3
```

Comment:
I don't like this static initializations, but since they have been used before: ok for this extension.

Reply:
fixed, there were a good buch of others from the previous code base

Reply:
was not my code :-) but what's the problem with them?

Reply:
e.g. IMO 

```
final Set<String> radiosWithoutManufacturer = Stream.of("", null).collect(Collectors.toSet());
```

is much better (and readable) than three lines within a `static` block. With OH3 (amd Java 11),we could even write

```
final Set<String>radiosWithoutManufacturer = Set.of("", null);
```

I think this should be refactored then.